tests/test_conferences.py

```diff
@@ -239,6 +237,66 @@ def test_addresses_from_111__a_double_c_d_e_g_x_y():
     assert expected == result['addresses']
 
 
+def test_addresses_from_111__a_c_d_e_g_x_y_three_address_parts():
+    schema = load_schema('conferences')
+    subschema = schema['properties']['addresses']
+
+    snippet = (
+        '<datafield tag="111" ind1=" " ind2=" ">'
+        '   <subfield code="a">10th Int. Conf. DICE2020: Spacetime - Matter - Quantum Mechanics</subfield>'
+        '   <subfield code="e">DICE2020</subfield>'
+        '   <subfield code="x">2020-09-14</subfield>'
+        '   <subfield code="y">2020-09-18</subfield>'
+        '   <subfield code="c">Castiglioncello , Tuscany, Italy</subfield>'
```

Comment:
I guess the extra space is on purpose ☺️

Reply:
Didn't even notice, it's from a real record